bigtable-hbase-replication/src/main/java/com/google/cloud/bigtable/hbase/replication/adapters/BigtableWALEntry.java

```diff
@@ -10,12 +10,10 @@
 public class BigtableWALEntry {
     private long timeStamp;
     private ArrayList<Cell> cells;
-    private String tableName;
```

Comment:
Why did we remove the table name? Do we have access to the table name in the IncompatibleMutationAdapter? It is important there. If we have to query CBT for the incompatible mutation, we need the table name.